.codeclimate.yml

```diff
@@ -24,12 +24,17 @@ checks:
   return-statements:
     config:
       threshold: 4
-  identical-code:
-    config:
-      threshold: 120
-  similar-code:
-    config:
-      threshold: 100
+# https://docs.codeclimate.com/docs/duplication
+plugins:
+  duplication:
+    enabled: true
+    config:
+      count_threshold: 3
+      languages:
+        typescript:
+          mass_threshold: 80
```

Comment:
Do we want to adjust this threshold up? Are all the duplications identified valid?

Reply:
At first glance it looks like they are valid enough. However, we may want to add specific rules to exclude some language constructions, for example, class/function initialization, filters etc. The overall amount of smells dropped from 190 to 129

Reply:
Would be nice if we can drop it even more in a subsequent pr